osgi-test/build.gradle

```diff
@@ -66,6 +66,8 @@ test {
   systemProperty 'ehcache.osgi.jar', project(':dist').jar.archivePath.getPath()
   systemProperty 'ehcache.osgi.jcache.version', parent.jcacheVersion
   systemProperty 'ehcache.osgi.slf4j.version', parent.slf4jVersion
+}.doFirst {
```

Comment:
That is sad. So you know if it's getting fixed? Maybe we should print a warning to tell people building the project that a part is missing